services/outputhost/messagecache.go

```diff
@@ -781,6 +786,8 @@ func (msgCache *cgMsgCache) manageMessageDeliveryCache() {
 					msgCache.numAcks = 0
 				}
 			}
+		case <-msgCache.cfgRefreshTicker.C:
```

Comment:
Yeah I was mainly worried about the calls to cassandra every 500ms (which is the redeliveryTicker) but since it is caching it, i will update to use the same ticker..